src/org.xtuml.bp.doc/WhatsNew/WhatsNew.html

```diff
@@ -12,43 +12,49 @@
   <![endif]-->
 </head>
 <body>
-<h1 id="whats-new-in-bridgepoint-6.0.0">What's New in BridgePoint 6.0.0</h1>
+<h1 id="whats-new-in-bridgepoint-6.2.0">What's New in BridgePoint 6.2.0</h1>
 <h2 id="release-highlights">Release Highlights</h2>
 <ul>
-<li>Marking Editor</li>
-<li>MASL</li>
-<li>Model Compilers</li>
-<li>Native platform builds (32 and 64 bit)</li>
+<li>MASL Editor</li>
+<li>MASL Flow</li>
+<li>OAL to MASL Converter</li>
 </ul>
 <hr />
-<h2 id="marking-editor">Marking Editor</h2>
-<p>BridgePoint now supports a general purpose marking editor. The editor relies on<br />
-feature markings specified by a model compiler architect. The user then provides<br />
-application-specific marking values.</p>
-<p><img src="marking_editor.png" alt="Marking Editor" /></p>
-<p>The BridgePoint documentation contains detailed information about the configuration<br />
-and use of the marking editor. See Help &gt; Help Contents &gt; BridgePoint UML Suite Help &gt; Reference &gt; User Interface &gt; Marking Editor.</p>
-<h2 id="masl">MASL</h2>
+<h2 id="masl-editor">MASL Editor</h2>
+<ul>
+<li>BridgePoint now supports a single-activity MASL editor (Figure 1). The editor<br />
```

Comment:
This is focused on technology.  It needs to focus on functionality.  Skip the discussion of the underlying file.  Focus on the signature, syntax highlighting and (the big) on rename refactoring (automatic reference maintenance).